standalone/pruntime/enclave/src/lib.rs

```diff
@@ -1634,6 +1637,56 @@ fn test_ink(_input: &Map<String, Value>) -> Result<Value, Value> {
     Ok(json!({}))
 }
 
+const INTERVAL: u64 = 30;
+const MAX_NUM: u128 = 65536*1024;
+
+fn is_prime(num: u128) -> bool {
+    let tmp = num - 1;
+    for i in tmp..=2 {
+        if num % i == 0 {
+            return false;
+        }
+    }
+
+    true
+}
+
+fn check_prime() {
+    for i in 2..MAX_NUM {
+        let _ = is_prime(i);
```

Comment:
Functions that have no side effects like this one would likely be able to be optimized away by the compiler. So the `check_prime` would become a NOP some day in the future along with the improving of rustc.